docs/examples/en/animations/CCDIKSolver.html

```diff
@@ -11,31 +11,94 @@ <h1>[name]</h1>
 
 		<p class="desc"> A solver for IK with <a href="https://sites.google.com/site/auraliusproject/ccd-algorithm"><em>CCD Algorithm</em></a>. <br /><br />
 		[name] solves Inverse Kinematics Problem with CCD Algorithm.
-		[name] is designed to work with [page:SkinnedMesh] loaded by [page:MMDLoader] but also can be used for generic [page:SkinnedMesh].
+		[name] is designed to work with [page:SkinnedMesh] but also can be used with [page:MMDLoader] or [page:GLTFLoader] skeleton.
 		</p>
 
+		<iframe id="scene" src="scenes/ccdiksolver-browser.html"></iframe>
```

Comment:
On tiny thing: Can you also include the iFrame and the updated code example in the `zh` version?

Reply:
Sorry what do you call `zh` version?

Reply:
Check out: https://threejs.org/docs/index.html#examples/zh/animations/CCDIKSolver

(The page is just not translated, yet.)

Reply:
got it! I will add an updated version of https://github.com/mrdoob/three.js/blob/dev/docs/examples/zh/animations/CCDIKSolver.html 👍

Reply:
added in https://github.com/mrdoob/three.js/pull/23449/commits/298a6f0c4f07ededdd72c66c25740cd95243ae3e